src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx

```diff
@@ -78,6 +92,18 @@ const EditQuizContainer = ({ questions, setQuestions }: QuizContentProps) => {
           value={description}
           variant={TextFieldVariantEnum.Standard}
         />
+        <AsyncAutocomplete
+          labelField='name'
+          onChange={onCategoryChange}
+          service={categoryService.getCategoriesNames}
+          sx={styles.categoryInput}
+          textFieldProps={{
+            label: t('myResourcesPage.quizzes.categoryDropdown'),
+            helperText: t('myResourcesPage.quizzes.categoryDropdownHelper')
```

Comment:
I guess, you don`t need this helper text, category field is not required and you can only choose values from options list, so, validation is redundant here

Reply:
but helperText only describes the category field, it doesn't do any validations. We need it as it is showed in figma design.